pkg/providers/cloudstack/cloudstack.go

```diff
@@ -364,6 +417,7 @@ func (p *cloudstackProvider) validateEnv(ctx context.Context) error {
 			return fmt.Errorf("unable to set %s: %v", eksaLicense, err)
 		}
 	}
+	p.execConfig = execConfig
```

Comment:
what's the point of the temporary `execConfig` variable? Why not just use `p.execConfig` throughout?

Reply:
Fixed